tools/bridge-deploy/Makefile

```diff
@@ -44,4 +44,8 @@ install: install-requirements compile
 $(VIRTUAL_ENV):
 	python3 -m venv $@
 
-.PHONY: install install-requirements test lint compile build format
+clean:
```

Comment:
Remove *build* and *node_modules* from the poa submodule as well?

Reply:
I chose calling `git submodule deinit`.